### ISSUE TITLE
Feature request: support Shizuku

### ISSUE THREAD

**Issue by sorz** [brevent/Brevent#81]
Hello,

I'm a user of [App Ops](https://play.google.com/store/apps/details?id=rikka.appops), which seems share an common idea with Brevent in terms of gaining privileges via adb. Recently, App Ops moved privileged adb API calls into [Shizuku](https://play.google.com/store/apps/details?id=moe.shizuku.privileged.api) [Manager](https://github.com/RikkaW/Shizuku), a separated app that play a gateway role between ordinary apps and adb:

> Shizuku Server is a process started by root or adb, therefore the process can invoke some privileged APIs that ordinary applications do not have permission to call, and ordinary applications can be invoked by interacting with the process. 
>Shizuku Manager is used to start Shizuku Server and manage the apps who can use this service.

I guess Shizuku Manager is designed for all kinds of apps who need adb privileges. (@RikkaW Am I right?

I wonder that if Brevent can adopt Shizuku. It will reduce manual operations on every boots if both apps are installed. (Any other pros and cons?

(I know little about adb hack and these apps, sorry for any misleading & unrealistic thinks.

Thanks.

**Comment by aviraxp** [comment]
Shizuku is inspired by Brevent, and uses some code of Brevent. I think we will never support it, as Shizuku does not provide all the API Brevent uses, and Brevent's implementation is better.

**Comment by liudongmiao** [comment]
@sorz Appops is a manager to control app ops, which is part of Android System. Brevent is a manager to control Brevent List and other Brevent's configurations, which is part of Brevent Server. Brevent need no Android permissions, and Brevent-Server is event-driven and is responsible to force-stop / standby other apps if other apps is started by user directly or by other apps. In earlier version, Brevent-server offer support for app ops, and removed later.

It's possible for appops to depend Brevent-Server, but it doesn't make any sense for Brevent to depend on other manager, as the actual working part is Brevent-Server, which shouldn't be an Android process, otherwise, users / other apps can stop Brevent-Server via Android API. And when you uninstall the appops, the modified appops will continue work if you don't revert. However, if you uninstall Brevent or reboot, Brevent has no other effect to your devices. This is why I removed the appops support.

I'm designing a simple hardware to active adb without pc, and will act as a manager for other apps which requires adb, but won't provide many services like brevent's api, which should be implemented by other apps directly (however, may offer some simple api).

I'm shamed if I'm shiziku manager's author, use codes and ideas of opensource projects like libsuperuser(and change package name), android support library, Brevent(some codes and ideas) without any mention. I released the source codes in wtfpl, and keep brevent-server close source since v0.9.

In previous, @RikkaW refused to mention Brevent and said it would violate play policy. But RikkaW violate play policy to show third party payment in appops.
